app/views/tag_migrations/index.html.erb

```diff
@@ -1,5 +1,5 @@
 <%= display_header title: 'Tag Migrations', breadcrumbs: ['Bulk Tagging'] do %>
```

Comment:
Sorry, only spotted this now. Should we rename the breakcrumbs to `Tag Searches` instead of `Bulk Tagging`?